t/after.t

```diff
@@ -0,0 +1,17 @@
+use strict;
+use warnings;
+
+use Test::More tests => 1;
+use File::Temp;
+use Rex::Commands;
+
+$::QUIET = 1;
+
+my $fh = File::Temp->new;
+
+task  foo => "asdfsadfasdf" => sub { return "yo" };
```

Comment:
Are there a way to make this fail faster? On my machine this test takes 14 seconds, while it basically just waits for the SSH connection to fail.